std_msgs/Header header

byte FIRST_GOAL_STARTED = 1
byte FIRST_GOAL_FINISHED = 3
byte DOCKING_FINISHED = 5
byte MOVING_BOX_STARTED = 19
byte MOVING_BOX_ENDED = 7
byte BOX_IS_ON_THE_ROBOT = 21
byte SECOND_GOAL_STARTED = 9
byte SECOND_GOAL_FINISHED = 11
byte ROBOT_FALLS = 0
byte BOX_FALLS = 2
byte ACTION

